feeds/src/state/ducks/aggregator/reducers.ts

```diff
@@ -1,9 +1,16 @@
+import 'core-js/stable/object/from-entries'
```

Comment:
I thought we only needed this in tests? This should be injected for us during web pack build